frontend/packages/otr/src/components/clerkInterpreter/overview/QualificationListing.tsx

```diff
@@ -111,11 +85,7 @@ export const QualificationListing = ({
                 </Text>
               </TableCell>
               <TableCell>
-                <div className="columns gapped-xs">
-                  <Text>
-                    {translateCommon(`examinationType.${q.examinationType}`)}
```

Comment:
En poistanut lokalisaatiotiedostoita näitä `examinationType`:n lokalisointeja. Ajattelin että voisivat olla käteviä vähintäänkin dokumentointimielessä. Mutta ehkä näiden merkitykset voisi olla hyvä siirtää muualle?

Reply:
Tää ok